modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/info/item/BlogsEntryInfoItemFields.java

```diff
@@ -117,6 +117,15 @@ public class BlogsEntryInfoItemFields {
 			InfoLocalizedValue.localize(
 				BlogsEntryInfoItemFields.class, "modified-date")
 		).build();
+	public static final InfoField<ImageInfoFieldType> previewImageInfoField =
```

Comment:
This new field is not present on any of the content of  BlogsEntryInfoItemFormProvider

If this is intentional, LGFM

Reply:
Hey @AliciaGarciaGarcia !!

I didn't want to change the BlogsEntryInfoItemFormProvider, because I don't now the implications of this change. Do you think we should add it?

Thanks!!!

Reply:
There is another way to use the new field `previewImageInfoField` if not from a `InfoItemFormProvider`? (honest question)

<img src="https://media2.giphy.com/media/dcScHZPUskMOJwyqjj/giphy.gif"/>

Reply:
Yeahh!!

You can see the full PR in [LPS-152905 Add Preview for blogs ](https://github.com/liferay-tango/liferay-portal/pull/2654).

We have created a new ContentDashboardAction with the previewImage of the blog: https://github.com/liferay-tango/liferay-portal/pull/2654/commits/603aef8cfabcb79a04d5b60c3a351c0c5ec30f90

Thanks!!

Reply:
so, you are creating the field "previewImage" to match the same field name with a WC etc. to simplify the access

If not, I did not understand it because you can use the coverimage and small image logic to obtain it.

Thanks for the reasoning

Reply:
Hey @AliciaGarciaGarcia !!

Exactly, after this change  we will read the previewImage from the content dashboard, so we don't need to implement a different logic for WC/Blogs/Files/...

Thanks!!